src/app/modules/timesheet/components/invoice-pdf/invoice-pdf.component.ts

```diff
@@ -26,16 +22,15 @@ export class InvoicePDFComponent implements OnInit {
 
   timesheet: Timesheet;
   local = 'fr';
-  workedTime: number;
   totalHT: number;
   totalTTC: number;
-  vat: number;
-  period: Date;
-  currency: string;
+  vatRate: number;
+  currencyCode: string;
+  workedTime: number;
 
   constructor(
-    private calendarManager: CalendarService,
-    private currencyService: MonetaryService,
+    public calendarService: CalendarService,
+    private monetaryService: MonetaryService,
```

Comment:
I my team when a value is private we begin it by a "_" in order to differentiate them to public vars